### PR TITLE
docs: Add Javascript examples for clickGesture and flingGesture

### PR DESCRIPTION
**Pull request by dopudj**
Add Javascript examples for two different Gestures since there are no code examples for JS

### CODE REVIEW

File: docs/android-mobile-gestures.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ driver.executeScript("mobile: clickGesture", ImmutableMap.of(
 driver.execute_script('mobile: clickGesture', {'x': 100, 'y': 100})
 ```
 
+```javascript
+// Javascript
```
Comment (mykola-mokhnach) [inline docs/android-mobile-gestures.md:109]: if it is WDIO then please put the framework name into the comment

File: docs/android-mobile-gestures.md [hunk 2]
```diff
@@ -172,6 +176,15 @@ boolean canScrollMore = (Boolean) ((JavascriptExecutor) driver).executeScript("m
 ));
 ```
 
+```javascript
+// Javascript
+await driver.executeScript('mobile: flingGesture', [{
+        elementId: element.elementId,
```
Comment (mykola-mokhnach) [inline docs/android-mobile-gestures.md:182]: Please use 4-space indent

File: docs/android-mobile-gestures.md [hunk 3]
```diff
@@ -105,6 +105,10 @@ driver.executeScript("mobile: clickGesture", ImmutableMap.of(
 driver.execute_script('mobile: clickGesture', {'x': 100, 'y': 100})
 ```
 
+```javascript
+// Javascript - Mocha
```
Comment (mykola-mokhnach) [inline docs/android-mobile-gestures.md:109]: Mocha is a general-purpose test runner framework. I meant the javascript wrapper over Appium, like WebdriverIO.

Comment (dopudj) [reply docs/android-mobile-gestures.md:109]: You are right, I'm using wdio, so is it ok if I change the line to :

`// Javascript - @wdio
`

or to write it somehow differently?

Comment (mykola-mokhnach) [reply docs/android-mobile-gestures.md:109]: yes, it works

**Review by eglitise — APPROVED** [review]

**Comment by jlipps** [comment]
Hi @dopudj , congrats, the Appium project wants to compensate you for this (and perhaps other) contribution(s) this month! Please reply to this comment mentioning me and sharing your [OpenCollective](https://opencollective.com) account name, so that we can initiate payment! Or let me know if you decline to receive compensation via OpenCollective. Thank you!

**Comment by dopudj** [comment]
Thanks @jlipps , it was a pleasure. Also thanks for sharing OpenCollective platform, I haven't used it before, my account username is [dopudj](https://opencollective.com/dopudj). I hope to be more involved in helping the community over time.

**Pull request merged**